src/Microsoft.AspNet.Mvc.Core/Filters/IFormatFilter.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Net.Http.Headers;
+using System;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// Implement this interface if you want to have your own implementation of FormatFilter. A FormatFilter decides 
+    /// what content type to use if the format is present in the Url. 
```

Comment:
I'd say something like like `A filter which produces a desired content type from the current request.`

The rest of the details here are good, but they belong in `<remarks>...`